terraform/environments/electronic-monitoring-data/lambdas_iam.tf

```
@@ -409,3 +409,61 @@ resource "aws_iam_role_policy_attachment" "update_log_table_get_log_s3_files" {
     role = aws_iam_role.update_log_table.name
     policy_arn = aws_iam_policy.get_log_s3_files.arn
 }
+
+# ------------------------------------------
+# output_file_structure_as_json_from_zip
+# ------------------------------------------
+
+resource "aws_iam_role" "output_fs_json_lambda" {
+  name = "output_fs_json_lambda"
+  assume_role_policy  = data.aws_iam_policy_document.lambda_assume_role.json
+  managed_policy_arns = ["arn:aws:iam::aws:policy/service-role/AWSLambdaBasicExecutionRole"]
+}
+
+data "aws_iam_policy_document" "output_fs_json_lambda_s3_policy_document" {
+  statement {
+    sid    = "S3Permissions"
```

```suggestion
    sid    = "S3PermissionsForUnzippingLambda"
```
better to be more specific here imo